project/ScalazBuild.scala

```diff
@@ -25,8 +25,7 @@ object Scalaz {
     "-Xlint:_,-type-parameter-shadow",
     "-Xsource:2.13",
     "-Ywarn-numeric-widen",
-    "-Ywarn-value-discard",
-    "-Xfatal-warnings"
```

Comment:
Why are you dropping fatal warnings?

Reply:
There were some warnings, thanks, I added them now

Reply:
I don't know how my comment disappear :D !

Reply:
@NeQuissimus  I added this plugin: 
```sbt

addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings"  % "3.0.0")
```